Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1092,68 +1102,21 @@ def disable_user(default_base_dn):
-    :return: The enabled user
     """
-    try:
-        user_profile = args.get("user-profile")
-        user_profile_delta = args.get('user-profile-delta')
-        iam_user_profile = IAMUserProfile(user_profile=user_profile, user_profile_delta=user_profile_delta)
-
-        ad_user = iam_user_profile.map_object(mapper_name=mapper_out)
-
-        # check it user exists and if it doesn't, create it
-        sam_account_name = ad_user.get("samaccountname")
-        if not sam_account_name:
-            raise DemistoException("User must have SAMAccountName")
-
-        user_exists = check_if_user_exists_by_samaccountname(default_base_dn, sam_account_name)
-
-        if not user_exists and create_if_not_exists:
-            return create_user_iam(default_base_dn, iam_user_profile, mapper_out)
-
-        dn = user_dn(sam_account_name, default_base_dn)
-
-        # modify user to enable account
-        modification = {
-            'userAccountControl': [('MODIFY_REPLACE', NORMAL_ACCOUNT)]
-        }
-
-        # Enable account and update with attributes if any
-        command_failed = False
-        modify_object(dn, modification)
-        if disabled_users_group_cn:
-            grp_dn = group_dn(disabled_users_group_cn, default_base_dn)
-            success = microsoft.removeMembersFromGroups.ad_remove_members_from_groups(conn, [dn], [grp_dn], True)
-            if not success:
-                command_failed = True
-                e = 'Failed to add user to {} group'.format(disabled_users_group_cn)
-                iam_user_profile.set_result(success=False,
-                                            error_message=e,
-                                            action=IAMActions.ENABLE_USER,
-                                            )
-        if not command_failed:
-            iam_user_profile.set_result(success=True,
-                                        email=ad_user.get('email'),
-                                        username=ad_user.get('name'),
-                                        action=IAMActions.ENABLE_USER,
-                                        details=ad_user,
-                                        active=True)
-
-        return iam_user_profile
-
-    except Exception as e:
-        iam_user_profile.set_result(success=False,
-                                    error_message=str(e),
-                                    action=IAMActions.ENABLE_USER,
-                                    )
-        return iam_user_profile
+    modification = {
+        'userAccountControl': [('MODIFY_REPLACE', NORMAL_ACCOUNT)]
+    }
+    modify_object(dn, modification)
+    if disabled_users_group_cn:
+        grp_dn = group_dn(disabled_users_group_cn, default_base_dn)
+        success = microsoft.removeMembersFromGroups.ad_remove_members_from_groups(conn, [dn], [grp_dn], True)
+        if not success:
+            raise Exception('Failed to remove user from {} group'.format(disabled_users_group_cn))
```

Comment:
is an error entry returned if the task fails? (also relevant to other exceptions raised from the iam commands)

Reply:
no... an error might be raised but if the command fails it will only return "False", it's like that in the other AD commands too